src/serializers/fields.rs

```diff
@@ -210,7 +211,28 @@ impl GeneralFieldsSerializer {
             // Check for missing fields, we can't have extra fields here
             && self.required_fields > used_req_fields
         {
-            Err(PydanticSerializationUnexpectedValue::new_err(None))
+            let required_fields = self.required_fields;
+            let field_name = extra.field_name.unwrap_or("<unknown field>").to_string();
```

Comment:
I think it's very possible for this to be `<unknown field>` if the value failing to serialize is the top-level model. Maybe add a test? Also I think in that case that `<unknown field>` is not helpful, perhaps just omitting the field name is better?